### ISSUE TITLE
Can't locate App/sdif/default.pm

### ISSUE THREAD

**Issue by eserte** [kaz-utashiro/sdif-tools#2]
On some of my smoker systems the t/01_run.t test fails:
```
App::sdif::default: Can't locate App/sdif/default.pm in @INC (you may need to install the App::sdif::default module) (@INC contains: /home/cpansand/.cpan/build/2017121103/App-sdif-3.2.2-1/lib /home/cpansand/.cpan/build/2017121103/App-sdif-3.2.2-1/blib/arch /home/cpansand/.cpan/build/2017121103/App-sdif-3.2.2-1/blib/lib /usr/perl5.18.4p/lib/site_perl/5.18.4/amd64-freebsd /usr/perl5.18.4p/lib/site_perl/5.18.4 /usr/perl5.18.4p/lib/5.18.4/amd64-freebsd /usr/perl5.18.4p/lib/5.18.4 .) at (eval 14) line 2.
BEGIN failed--compilation aborted at (eval 14) line 2.
 at /usr/perl5.18.4p/lib/site_perl/5.18.4/Getopt/EX/Loader.pm line 73.

#   Failed test at t/01_run.t line 12.
#          got: '512'
#     expected: '0'
# Looks like you failed 1 test of 1.
t/01_run.t ...... 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/1 subtests 
```

**Comment by eserte** [comment]
I see this problem also with App-sdif-3.2.3

**Comment by kaz-utashiro** [comment]
It seems that happens on the system which does not return "No such file or directory" in $! when an error ENOENT occurred.  It is a problem of Getopt::EX::Loader.pm, and fixed in v1.3.1 to use $!{ENOENT} instead.  sdif-3.2.3 did not declare the correct requisite.  I hope it works fine now, but don't have such system in hand.

**Comment by eserte** [comment]
For the record: to reproduce the problem one probably needs to use a non-English locale AND perl < 5.22.

But it seems that the problem does not happen anymore with 3.3.1.
